Functions.Templates/Templates/McpToolTrigger-CSharp-Isolated/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": {
+    "text": "MCP Tool trigger",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "8000"
```

Comment:
Set to 8000 because it seemed like the an ID that was not used for our existing templates, but I don't know if this is right?